packages/terra-form-select/src/shared/_SearchResults.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  /**
+   * Content to display when search results are found.
+   */
+  searchResultContent: PropTypes.node,
+};
+
+const SearchResults = ({ searchResultContent }) => (
+  <div role="option" aria-selected="false">
```

Comment:
I think we need to add styles for padding similar to no results component :
```css
    padding: var(--terra-form-select-search-results-padding, 0.28571rem 0.714286rem);
    word-break: break-word;
    word-wrap: break-word;
```

Reply:
Updated.